lib/testobject.js

```
@@ -181,6 +181,11 @@ TestObject.fetchAppium = async function (driverName, driverUrl) {
   await exec('git', ['clone', 'https://github.com/appium/appium.git'], {cwd: tempdir});
   logger.debug(`Cloned appium at ${tempdir}`);
 
+  // Enforce 'https://' over 'git@' for driverUrl. 'git@' is unauthorized in CI.
+  if (driverUrl.indexOf('git')) {
+    logger.error(`Error in driver url: ${driverUrl}. Use https instead.`);
```

I assume we should throw an exception if it is an error. Otherwise warn